src/components/TopNav/NavMenu.js

```diff
@@ -8,26 +8,24 @@ import NavigationMenu from 'material-ui/svg-icons/navigation/menu'
 import IconButton from 'material-ui/IconButton'
 import * as auth from 'services/authNService'
 
-export const mapStateToProps = (state) => {
-  return {
-    alias: state.auth.userAlias
-  }
-}
 
-export const NavMenu = ({dispatch, alias}) =>
-  <IconMenu
+
+
+export const NavMenu = ({ dispatch }) =>
+<IconMenu
+
     iconButtonElement={<IconButton><NavigationMenu /></IconButton>}
     anchorOrigin={{horizontal: 'left', vertical: 'bottom'}}
     targetOrigin={{horizontal: 'left', vertical: 'top'}}
 >
-    <MenuItem primaryText={alias} />
-    <MenuItem primaryText={<Link to='/search' >Incident Search</Link>} />
-    <MenuItem primaryText={<Link to='/' onClick={() => dispatch(auth.logOut)}>LogOut</Link>} />
-    <MenuItem primaryText={<Link to='/debug' >Debug</Link>} />
-  </IconMenu>
+
+    <MenuItem primaryText={<Link to="/search" >Incident Search</Link>} />
+    <MenuItem primaryText={<Link to="/" onClick={() => dispatch(auth.logOut)}>LogOut</Link>} />
```

Comment:
This all looks good.  I approve the commit.  Please feel free to merge.  

On a side note, I am curious about the switch from single quotes to double quotes.  Are we doing it that way now?  Was it necessary for some reason?  Or is it personal preference?  :)

Reply:
This breaks the linter. Should be single quotes.